compiler/rustc_query_impl/src/lib.rs

```diff
@@ -26,7 +26,7 @@ use rustc_span::Span;
 #[macro_use]
 mod plumbing;
 pub use plumbing::QueryCtxt;
-use rustc_query_system::query::*;
+pub use rustc_query_system::query::*;
```

Comment:
Why is this `pub` required?

Reply:
We need to use `try_collect_active_jobs` and `deadlock` in `rustc_interface::util::handle_deadlock`. We can add `pub use rustc_query_system::query::{deadlock, QueryContext}` instead of pub the all thing here.